Documentation/Signoffs/Camera_Software_System.md

```diff
@@ -1,37 +1,87 @@
 # Camera Software System
-## Functionality
-
+## Functionality 
+The camera software system will be reponsible for controlling the camera hardware system, ensuring that the system is able to take an image of an unmanned aircraft system (UAS) in-flight or the UAS pilot and save it to the database for usage by campus police.
 ## Constraints
 | No.| Contraint | Origin |
 | -- | --------- |--------|
-|  1 |           |        |              
-|  2 |           |        |                          
-|  3 |           |        |              
+|  1 | The system shall not account for light levels and/or obstructions to the view of the UAS or control station when determining whether to take a picture | Stakeholder Constraint | 
```

Comment:
find the most relevant constraint then attach to the end. this is what will happen...but this wont happen.

Reply:
light levels is a fuzzy term. may not be a software completely. the camera has its own light source so may need to say we wont do the flash in the hardware constraint

Reply:
contraint: will only succeed in predicting motion in these specific cases: straight line.  

accuracy constrain on output

opperating system constraint
how often do we have to output things to camera to keep things moving (analysis)